walletnode/services/artworkregister/config.go

```diff
@@ -25,7 +25,6 @@ type Config struct {
 	// internal settings
 	connectToNextNodeDelay time.Duration
 	acceptNodesTimeout     time.Duration
-	connectTimeout         time.Duration
```

Comment:
Why do we remove `connectTimeout` here ?

Reply:
this was added at a different place